cmd/snap-repair/testkeys.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build withtestkeys
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/asserts/systestkeys"
+)
+
+func init() {
+	// when built with testkeys enabled, trust the TestStoreAccountKey
+	trustedRepairRootKeys = append(trustedRepairRootKeys, systestkeys.TestStoreAccountKey.(*asserts.AccountKey))
```

Comment:
my preference would be to follow the pattern for the other keys and have a dedicated

TestRepairRootAccountKey

given that it needs to be shared across packages it can have be defined in systestkeys

Reply:
alright I have the separate root of trust key working again, I did end up using a different approach to generate the key, I couldn't quite get all the assertstests helpers working, so I opted for this: https://gist.github.com/anonymouse64/f644d32f8fc24b98399cf2cb36891931 which seemed to work so :shrug: